_unittest/test_00_EDB.py

```diff
@@ -520,3 +520,7 @@ def test_67_add_void(self):
         void = self.edbapp.core_primitives.create_path(path, layer_name="TOP",
                                                        width="0.1mm")
         assert self.edbapp.core_primitives.add_void(plane, void)
+
+
+    def test_68_flip_layer_stackup(self):
+        assert self.edbapp.core_stackup.flip_stackup()
```

Comment:
```suggestion
        assert self.edbapp.core_stackup.flip_stackup()
        
```
New line at the end of the file is missing. Style check will complain :)

Reply:
thanks